pkg/app/ops/insightcollector/collector.go

```diff
@@ -13,3 +13,361 @@
 // limitations under the License.
 
 package insightcollector
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/pipe-cd/pipe/pkg/filestore"
+
+	"github.com/pipe-cd/pipe/pkg/insightstore"
+
+	"go.uber.org/zap"
+
+	"github.com/pipe-cd/pipe/pkg/datastore"
+	"github.com/pipe-cd/pipe/pkg/model"
```

Comment:
nit: import paths should be ordered as:

```
standard packages

3rd party packages

our packages
```

I know the Jetbrains' import paths organization is terrible.